src/endpoints/performTotpVerification.ts

```diff
@@ -2,7 +2,7 @@ import { requestApi } from '../requestApi';
 
 interface PerformTotpVerification {
     login: string;
-    otp: string;
+    otp: number;
```

Comment:
i think it's still a string no?

Reply:
Main branch was not up to date. Should be better now